index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>index</title>
+    <script src="index.js" defer></script>
+    <link rel="stylesheet" href="sanitize.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <header class="title__container">
+        <h1 class="title">NoteKeeper</h1>
+    </header>
+    <main class="main__container">
+    <section class="form__container card__container">
+        <h2>New Note Form</h2>
+        <form method="post" action="">
```

Comment:
`method` and `action` are not necessary. Why were they implemented?

- [ ] ⚠️ Remove unused attributes.